### PR TITLE
feat: [CCM-7235]: New design for AS summary page

### PR DESCRIPTION
**Pull request by rishik30**
##### Screenshots:

Screenshot 2022-06-30 at 6 23 39 PM
Screenshot 2022-06-30 at 6 23 49 PM
Screenshot 2022-06-30 at 6 25 01 PM
Screenshot 2022-06-30 at 6 25 10 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/75-ce/components/COGatewayList/charts/RuleSavingsPieChart.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    plotOptions: {
+      pie: {
+        allowPointSelect: false,
+        cursor: 'pointer',
+        dataLabels: {
+          enabled: false
+        },
+        showInLegend: false,
+        size: '200%',
+        borderColor: disable ? '#F3F3FA' : ''
```
Comment (jenil-harness) [inline src/modules/75-ce/components/COGatewayList/charts/RuleSavingsPieChart.tsx:58]: Do we have variable for this color?

File: src/modules/75-ce/components/COGatewayList/charts/RuleSavingsPieChart.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  let secondaryColor = '#E4F7E1'
+  if (disable) {
+    primaryColor = '#6B6D85'
+    secondaryColor = '#D9DAE5'
+  } else if (savings <= 20) {
+    primaryColor = '#B41710'
+    secondaryColor = '#FCEDEC'
+  } else if (savings > 20 && savings <= 50) {
+    primaryColor = '#ff5310'
+    secondaryColor = '#fff0e6'
```
Comment (jenil-harness) [inline src/modules/75-ce/components/COGatewayList/charts/RuleSavingsPieChart.tsx:28]: Possible to use css vars?

File: src/modules/75-ce/common/InstanceStatusIndicator/InstanceStatusIndicator.tsx [hunk 3]
```diff
@@ -6,32 +6,39 @@
...
 
 export const RunningStatusIndicator = () => {
+  const { getString } = useStrings()
   return (
     <Layout.Horizontal className={css.instanceStatusIndicator}>
       <Icon name={'play'} />
-      <span style={{ color: '#42ab45' }}>Running</span>
+      <span style={{ color: Color.GREEN_600, textTransform: 'capitalize' }}>
+        {getString('ce.co.ruleState.running')}
+      </span>
```
Comment (jenil-harness) [inline src/modules/75-ce/common/InstanceStatusIndicator/InstanceStatusIndicator.tsx:21]: use `<Text>` component here

File: src/modules/75-ce/common/InstanceStatusIndicator/InstanceStatusIndicator.tsx [hunk 4]
```diff
@@ -6,32 +6,39 @@
...
   )
 }
 
 export const StoppedStatusIndicator = () => {
+  const { getString } = useStrings()
   return (
     <Layout.Horizontal className={css.instanceStatusIndicator}>
       <Icon name={'stop'} />
-      <span style={{ color: '#DA291D' }}>Stopped</span>
+      <span style={{ color: Color.RED_600 }}>{getString('ce.co.ruleState.stopped')}</span>
```
Comment (jenil-harness) [inline src/modules/75-ce/common/InstanceStatusIndicator/InstanceStatusIndicator.tsx:31]: `Text` component

File: src/modules/75-ce/common/InstanceStatusIndicator/InstanceStatusIndicator.tsx [hunk 5]
```diff
@@ -6,32 +6,39 @@
...
   )
 }
 
 export const CreatedStatusIndicator = () => {
+  const { getString } = useStrings()
   return (
     <Layout.Horizontal className={css.instanceStatusIndicator}>
       <Icon name={'full-circle'} size={10} />
-      <span style={{ color: '#42ab45' }}>Created</span>
+      <span style={{ color: Color.GREEN_500 }}>{getString('created')}</span>
```
Comment (jenil-harness) [inline src/modules/75-ce/common/InstanceStatusIndicator/InstanceStatusIndicator.tsx:41]: `Text` component

File: src/modules/75-ce/components/COGatewayList/COGatewayCumulativeAnalytics.tsx [hunk 6]
```diff
@@ -94,170 +102,195 @@ const COGatewayCumulativeAnalytics: React.FC<COGatewayCumulativeAnalyticsProps>
...
-              <Layout.Vertical spacing={'medium'} style={{ paddingBottom: 'var(--spacing-large)' }}>
-                <Layout.Horizontal flex={{ justifyContent: 'flex-start', alignItems: 'center' }} spacing="small">
-                  <img src={grayLeaf} width={20} />
+                  {loadingData ? (
+                    <Icon name="spinner" size={24} color="blue500" />
+                  ) : (
+                    <>
+                      {hasData ? (
+                        <Text font={{ variation: FontVariation.H2 }} color={Color.GREY_700}>
+                          ${(Math.round(data?.total_savings as number) * 100) / 100}
```
Comment (jenil-harness) [inline src/modules/75-ce/components/COGatewayList/COGatewayCumulativeAnalytics.tsx:216]: `formatCost` util?

File: src/modules/75-ce/components/COGatewayList/COGatewayCumulativeAnalytics.tsx [hunk 7]
```diff
@@ -94,170 +102,195 @@ const COGatewayCumulativeAnalytics: React.FC<COGatewayCumulativeAnalyticsProps>
...
-                    useRichText
-                    vars={{ value: getEmissionsValue(_defaultTo(data?.total_cost, 0)) }}
-                  />
-                </Text>
-              </Layout.Vertical>
+                  {loadingData ? (
+                    <Icon name="spinner" size={24} color="blue500" />
+                  ) : (
+                    <Text font={{ variation: FontVariation.H3 }} color={Color.GREY_500}>
+                      ${(Math.round(data?.total_cost as number) * 100) / 100}
```
Comment (jenil-harness) [inline src/modules/75-ce/components/COGatewayList/COGatewayCumulativeAnalytics.tsx:248]: `formatCost` util

File: src/modules/75-ce/components/COGatewayList/COGatewayList.tsx [hunk 8]
```diff
@@ -131,26 +132,77 @@ interface SortByObjInterface {
...
+
   return (
-    <Text lineClamp={3} color={tableProps.row.original.disabled ? textColor.disable : Color.GREY_500}>
-      {tableProps.value} mins
-    </Text>
+    <Layout.Vertical spacing={'small'}>
+      <Text lineClamp={3} color={Color.GREY_1000}>
+        {getString('ce.co.ruleDetailsHeader.idleTime')}
+        {' : '}
+        {tableProps.value} min
```
Comment (jenil-harness) [inline src/modules/75-ce/components/COGatewayList/COGatewayList.tsx:163]: Move to string constants

**Review by jenil-harness — COMMENTED** [review]
Have a look

File: src/modules/75-ce/components/COGatewayList/charts/RuleSavingsPieChart.tsx (hunk 1 above)
Comment (rishik30) [reply src/modules/75-ce/components/COGatewayList/charts/RuleSavingsPieChart.tsx:58]: variables are not working inside the config

File: src/modules/75-ce/components/COGatewayList/charts/RuleSavingsPieChart.tsx (hunk 2 above)
Comment (rishik30) [reply src/modules/75-ce/components/COGatewayList/charts/RuleSavingsPieChart.tsx:28]: variables are not working inside the config

**Review by jenil-harness — APPROVED** [review]

**Review by jyotiarora-harness — APPROVED** [review]

**Pull request merged**